src/lightning_app/frontend/__init__.py

```diff
@@ -1,6 +1,6 @@
 from lightning_app.frontend.frontend import Frontend
-from lightning_app.frontend.panel import PanelFrontend
+from lightning_app.frontend.panel import AppStateWatcher, PanelFrontend, watch_app_state
 from lightning_app.frontend.stream_lit import StreamlitFrontend
 from lightning_app.frontend.web import StaticWebFrontend
 
-__all__ = ["Frontend", "PanelFrontend", "StaticWebFrontend", "StreamlitFrontend"]
+__all__ = ["Frontend", "PanelFrontend", "StaticWebFrontend", "StreamlitFrontend", "AppStateWatcher", "watch_app_state"]
```

Comment:
Can move `watch_app_state` to private. Maybe `AppStateWatcher` too.

Reply:
Kept `AppStateWatcher` as mentioned in docs